### PR TITLE
 PT-579 Integrate Tidepool v0.8.0

### PR DESCRIPTION
**Pull request by yvon-dblg**
Fix of the PR#9  https://github.com/mdblp/message-api/pull/9

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 Tidepools message API for adding context to diabetes data
 
-## 0.1.0
-
+## 0.2.0
 ### Changed
+- [PT-579] Integrate Tidepool [v0.8.0](https://github.com/tidepool-org/message-api/releases/tag/v0.8.0)
 
+## 0.1.0
```
Comment (CedricPerotto) [inline CHANGELOG.md:9]:
```suggestion
## 0.1.0 - 2019-07-30
```

File: Dockerfile [hunk 2]
```diff
@@ -8,9 +8,8 @@ WORKDIR /app
 COPY package.json .
 
 RUN apk add --no-cache --virtual .build-dependencies git && \
```
Comment (CedricPerotto) [inline Dockerfile:10]: With the updated package.json, npm install now needs python & make (just like in the previous PR, I think it is linked to user-api-client version change ...)
```suggestion
RUN apk add --no-cache --virtual .build-dependencies git python make && \
```

**Review by CedricPerotto — CHANGES_REQUESTED** [review]
Dockerfile update (python & make needed to run npm install... you already saw that) 

You can update the changelog (release date) in your next commit. I think it is ready for release.

**Comment by yvon-dblg** [comment]
Fix in PR https://github.com/mdblp/message-api/pull/11

**Pull request closed without merging**